tests/tool/Integration.Tests/IntegrationScenarios/AspNetSample/csharp/Upgraded/Views/Home/Index.cshtml

```diff
@@ -15,7 +14,7 @@
     <div class="col-md-4">
         <h2>Getting started</h2>
         @{ 
-            var x = new Microsoft.AspNetCore.Mvc.ActionResult();
+            var x = new ActionResult();
```

Comment:
seems like a behavioral change. Would expect a correlating change to `_ViewImports.cshtml` to make this work

Reply:
This is expected because of Razor magic I'm afraid :( When ASP.NET Core builds its Razor pages, it automatically includes a few namespaces. One of them is Microsoft.AspNetCore.Mvc.